### ISSUE TITLE
Feature request: Text Size in settings, and in comments of eg. Youtube

### ISSUE THREAD

**Issue by Giesa-lab** [futo-org/grayjay-android#3456]
### Use Cases

So this applies to (in my case) Grayjay version 391 with Youtube plugin version 356, but all versions and plugins for other services have the same persistent issue. The very tiny text, as seen on an android phone or even a larger tablet, is impossible to read except by using an inconvenient physical magnifying glass. 

This is because it is impossible to Zoom in the Grayjay App with two finger gesture, as in Chrome browser, ot indeed most other Android Apps. There are zero options to set text size in either the App itself, or for any of the plugins either. This is a major oversight, and frankly I blame the developers for no doubt composing these Apps and Plugins on very large display screens where this issue isn't apparent. Just try to read the text in "settings" on an ordinary tiny mobile phone screen, that cannot be zoomed with two finger gesture. Quite apart from the tiny size, the text is in gray on black in the Dark mode, and this makes readability even worse. 

Again, it's impossible to copy and paste ANY text from the comments of ANY plugin either (so as to paste into a notepad or email) and this is another oversight as well. Please give consideration to allow two finger zoom for the app in Android permissions, as there is no extant setting to do so, anywhere in the App, or in its Android settings either.

### Proposal

Allow two finger zoom for entire App window in the code somehow.
Make a setting to adjust text size in the plugin comments sections, and in the App sections themselves.

**Comment by defiling9046** [comment]
cc
#292

**Comment by Giesa-lab** [comment]
I'm adding this workaround for the tiny text issue, but I'd still like to be able to copy and paste text of the public comments in YouTube and Rumble etc.... 

*** WORKAROUND TO READ TINY TEXT ANYWHERE IN GRAYJAY (ANY VERSION) *** 

See attached screenshots... [edit - added 2 further screenshots showing impressive 6X zoom]
  
1. Go to Android Settings menu.
2. Search for "magnification"
3. Open magnification settings  and switch on "Triple Tap"
4. Close settings
5. Open Grayjay App Homepage
6. Triple Tap on any Blank area of the window (top right area is good)
7. Whole window magnifies about double size
8. Move viewed window using two finger gestures
9. *Now you CAN pinch/spread to zoom larger, remembers last zoomed size on exit
10. Triple tap again to return to normal size and exit magnifier 

Again I'm sure there's akso a way to easily allow some permission to do two finger pinch zooming by allowing this standard option switch in the App, but the App has to request this permission, and all the window zooming is done by Android system, I'd adumbrate  🤔

<img width="1918" height="1080" alt="Image" src="https://github.com/user-attachments/assets/e3aa24e7-87cf-4dd7-a305-0f453c5cde9b" />
<img width="1918" height="1120" alt="Image" src="https://github.com/user-attachments/assets/4a1d2bca-4585-48a8-812b-031b454ee942" />
<img width="1920" height="1200" alt="Image" src="https://github.com/user-attachments/assets/8e34de10-5171-4822-b6ec-33b5ea413d06" />
<img width="1918" height="1136" alt="Image" src="https://github.com/user-attachments/assets/eeef7d67-5156-4e6a-a95c-4c95574cf805" />
<img width="586" height="907" alt="Image" src="https://github.com/user-attachments/assets/b5b6e5c2-97f3-47a8-bea9-4d1a41e582c5" />
<img width="1918" height="1073" alt="Image" src="https://github.com/user-attachments/assets/7ec84476-569f-4532-92fd-302450704f3a" />
<img width="1920" height="1200" alt="Image" src="https://github.com/user-attachments/assets/b29b830d-88be-48fe-8935-b9b4bcf5d86d" />
